docs/usage.md

```diff
@@ -129,209 +133,267 @@ Please note the following requirements:
 
 ### `--use_z_ions`
+
 Include z ions into the peptide spectrum matching
 
 ### `--fdr_threshold`
+
 Specify the false discovery rate threshold at which peptide hits should be selected. (eg. 0.01)
 
 ### `--fdr_level`
+
 Specify the level at which the false discovery rate should be computed. 'peptide-level-fdrs' is recommended. ('peptide-level-fdrs', 'psm-level-fdrs', 'protein-level-fdrs')
 
 ### `--number_mods`
+
 Specify the maximum number of modifications that should be contained in a peptide sequence match. (eg. 3)
 
 ### `--num_hits`
+
 Specify the number of hits that should be reported for each spectrum. (eg. 1)
 
 ### `--digest_mass_range`
+
 Specify the mass range that peptides should fullfill to be considered for peptide spectrum matching. (eg. 800:2500)
 
 ### `--pick_ms_levels`
+
 If one ms level in the raw ms data is not centroided, specify the level here. (eg. 2)
 
 ### `--run_centroidisation`
+
 Choose whether the specified ms_level in pick_ms_levels is centroided or not. ("True", "False")
 
 ### `--prec_charge`
+
 Specifiy the precursor charge range that peptides should fullfill to be considered for peptide spectrum matching. (eg. "2:3")
 
 ### `--activation method`
+
 Specify which fragmentation method was used in the MS acquisition ('ALL', 'CID', 'ECD', 'ETD', 'PQD', 'HCD', 'IRMPD')
 
 ### `--enzyme`
+
 Specify which enzymatic restriction should be applied ('unspecific cleavage', 'Trypsin', see OpenMS enzymes)
 
 ### `--fixed_mods`
+
 Specify which fixed modifications should be applied to the database search (eg. '' or 'Carbamidomethyl (C)', see OpenMS modifications)
 
 ### `--variable_mods`
+
 Specify which variable modifications should be applied to the database search (eg. 'Oxidation (M)', see OpenMS modifications)
 
 ### `--max_rt_alignment_shift`
+
 Set a maximum retention time shift for the linear rt alignment
 
 ### `--spectrum_batch_size`
+
 Size of Spectrum batch for Comet processing (Decrease/Increase depending on Memory Availability)
 
 ### `--skip_decoy_generation`
+
 If you want to use your own decoys, you can specify a databaset that includes decoy sequences. However, each database entry should keep the prefix 'DECOY_'.
-One should consider though that this option will then prevent to append variants to the database and if not using reversed decoys the subset refinement FDR option will not work. 
+One should consider though that this option will then prevent to append variants to the database and if not using reversed decoys the subset refinement FDR option will not work.
 
 ## Optional binding prediction
+
 ### `--predict_class_1`
+
 Set flag depending on whether MHC class 2 binding predictions using the tool mhcflurry should be run. (Check whether your alleles are supported by mhcflurry)
 
 ### `--predict_class_2`
+
 Set flag depending on whether MHC class 1 binding predictions using the tool mhcnugget should be run. (Check whether your alleles are supported by mhcnugget)
```

Comment:
Should be class 2 and could also reference the supported class 2 alleles file again.